sdk/include/ds/pointer.h

```diff
@@ -104,7 +104,7 @@ namespace ds::pointer
 
 			__always_inline Pointer<T> &operator=(Pointer const &p)
 			{
-				ref = p.ref;
+				ref = (T *){p.ref};
```

Comment:
Doesn't this trigger a clang-tidy warning about C-style casts?

Reply:
I’ll double check tomorrow but I didn’t see one when I ran it. It’s intended to be a direct-list-initialization, not a C-style cast.

Reply:
Hmm.  I find that less readable than the original, but maybe that's just me.

Reply:
How would you feel about `ref = static_cast<T*>(p.ref)` or `*&ref = p.ref`?

Reply:
Switched to `static_cast`